Updating admin password on Clusters with CRDB (#1474)
RedisLabs/redislabs-docs: content/rs/security/admin-console-security/user-security.md

Here, number_of_min is the number of minutes after which sessions will time out.

## Updating Administrator password on Clusters with Active-Active databases

> Active-Active databases are relying on the Administrator credentials for management operations of the Active-Active database.

Reviewer: ```suggestion
Active-Active databases are relying on the administrator credentials for management operations of the Active-Active database.
```

When updating an administrator user password on a cluster with Active-Active databases please follow these steps:
1. From the user management page, update the administrator user password on the cluster/s you wish to update.

> 1. Go to each participating cluster and for each Active-Active database update the Admin user/s credentials that have you changed on step 1.

Reviewer: ```suggestion
1. Go to each participating cluster and for each Active-Active database update the admin user credentials that have you changed with the last step. 
```

> Do not perform any management operation on the databases between these two steps.

Reviewer: ```suggestion
{{< warnging> }}Do not perform any management operation on the databases between these two steps. {{< /warning >}}
```

Here, number_of_min is the number of minutes after which sessions will time out.

> ## Updating Administrator password on Clusters with Active-Active databases

Reviewer 2: ```suggestion
## Update Active-Active cluster admin credentials
```

> Active-Active databases are relying on the administrator credentials for management operations of the Active-Active database.

Reviewer 2: ```suggestion
Active-Active databases use administrator credentials to manage operations for Active-Active database.
```

> When updating an administrator user password on a cluster with Active-Active databases please follow these steps:

Reviewer 2: ```suggestion
To update the administrator user password on a cluster with Active-Active databases:
```

> 1. From the user management page, update the administrator user password on the cluster/s you wish to update.

Reviewer 2: ```suggestion
1. From the user management page, update the administrator user password on the clusters you wish to update.
```

> 1. Go to each participating cluster and for each Active-Active database update the admin user credentials that have you changed with the last step. 

Reviewer 2: ```suggestion
1. For each participating cluster _and_ each Active-Active database, update the admin user credentials to match the changes in Step 1. 
```

> {{< warnging> }}Do not perform any management operation on the databases between these two steps. {{< /warning >}}

Reviewer 2: ```suggestion
{{< warning> }}Do not perform any management operation on the databases until these steps are complete. {{< /warning >}}
```
